doc/how_to_introduce_breaking_changes.md

```diff
@@ -0,0 +1,198 @@
+# How to introduce Breaking Changes in service command
+
+Azure CLI has bi-annual breaking change releases coinciding with Microsoft **Build** and **Ignite**. Limiting breaking changes to twice a year provides a stable experience for customers while being able to keep up to date with the latest versions of Azure CLI and plan accordingly for announced breaking changes.
+
+## Breaking Changes in Azure CLI
+
+A breaking change refers to a modification that disrupts backward compatibility with previous versions. The breaking changes could cause a customer's script or automation written in a previous version to fail.
+
+The common examples of breaking changes include:
+* Modifying the names of parameters/commands.
+* Modifying the input logic of parameters.
+* Modifying the format or properties of result output.
+* Modifying the current behavior model.
+* Adding additional verification that changes CLI behavior.
+
+To mitigate the impact of breaking changes, Azure CLI delays breaking changes and coordinates half-yearly **Breaking Change Releases** that bundle multiple breaking changes together. This approach helps users plan ahead and adapt to the modifications effectively.
+
+### Breaking Change Window
+
+The breaking change window *allows* for service command breaking changes. When a Pull Request is merged during this sprint, it will be included in the next Breaking Change Release.
+
+The timing of the breaking change window in Azure CLI aligns with [Microsoft Build](https://build.microsoft.com/) and [Microsoft Ignite](https://ignite.microsoft.com/). You could find the next Breaking Change Release plan in our [milestones](https://github.com/Azure/azure-cli/milestones).
+
+> If you would like to release ad-hoc breaking changes, reach out to the CLI team to provide an explanation for the necessity of these changes. The exceptions can be provide in the following cases:
```

Comment:
my 2 cents here:
I personally like to rephrase this part and tie with why/what we build breaking change window, mentioned in the top.
Making the request easy will likely get us difficult to hold the door.


_It's highlighted that we usually do not allow breaking changes check-ins within non-breaking-change window, based on what we stated above for consistency and stable tooling experience.
something unexpected can happen. We understand and would like to help out. There would be high-graded justifications required to provide the info Azure CLI can access.
p.s.
Noting that providing the required info for assessment does not mean it will be assured to be greenlighted for breaking changes. Team will still make the decision based on the overall impact._

Reply:
I agree with that. I have replaced the parts about the ad-hoc introduction of breaking changes. Kindly review the modified content and provide your feedback.